packages/components/src/components/post-popovercontainer/post-popovercontainer.scss

```
@@ -0,0 +1,52 @@
+@use 'sass:meta';
+@use 'sass:math';
+@use '@swisspost/design-system-styles/variables/color';
+@use '@swisspost/design-system-styles/variables/commons';
+@use '@swisspost/design-system-styles/variables/spacing';
+@use '@swisspost/design-system-styles/mixins/color' as color-mx;
+@use '@swisspost/design-system-styles/mixins/elevation' as elevation-mx;
+@use '@swisspost/design-system-styles/functions/color' as color-fn;
+
+// Puts polyfilled styles in a separate layer so they are easy to override
+// Can be removed as soon as popover is supported by all major browsers
+// https://caniuse.com/?search=popover
+@layer polyfill {
+  @include meta.load-css('@oddbird/popover-polyfill/dist/popover.css');
+}
+
+div {
+  @include elevation-mx.elevation('elevation-3');
+
+  position: fixed;
+  z-index: commons.$zindex-popover;
+
+  width: max-content;
+  max-width: 30ch;
+  margin: 0;
+  padding: 0;
+
+  color: var(--post-contrast-color);
+  background-color: rgb(var(--post-bg-rgb));
+  border-color: transparent; // Keeping the default border for HCM
+  border-radius: commons.$border-radius;
+
+  // Keeps the little arrow visible
+  overflow: visible;
+}
+
+.arrow {
+  $arrow-size: math.div(spacing.$spacer, math.sqrt(2));
+  position: absolute;
+  // Diagonale of 16px -> 1rem -> 1/1.41 = ~0.7
+  // https://www.omnicalculator.com/math/square-diagonal?c=CHF&v=hide:0,diagonal:16!cm
+  width: $arrow-size;
+  height: $arrow-size;
+  background-color: inherit;
+  rotate: 45deg;
+  pointer-events: none;
+  z-index: -1;
+
+  // High contrast mode borders
```

```suggestion
  // Create transparent border to be styled by and for the high contrast mode
```
Maybe, it's a bit more explicit for people that are not aware.